packages/frame/src/vars/index.js

```diff
@@ -0,0 +1,13 @@
+export const breakpoints = {
```

Comment:
Are these breakpoints also worthy of core?  Or perhaps something to do later?

Are they general enough to be standardized, where we want other devs to use them for other things? If so, adding a card to put them into core seems appropriate.

Reply:
yes eventually we'll move then into core

cc @sweeting do we think this is a good initial list for core